src/reducers/listBooksReducer.js

```diff
@@ -0,0 +1,14 @@
+export default function selectedBookReducer(state = [], action) {
+
+    switch (action.type) {
+        case "LIST_BOOKS":
+            state= {...state, books: action.payload};
```

Comment:
it's useful, to keep a template for state manipulation to avoid forgetting to clone the object and edit the updated part only.
but here for both reducers, we can just return the action payload as the new state, like `state= {books: action.payload};` here for instance,
This will save extra processing, still minimal, so not a big deal.

Reply:
Done